src/components/SideNavBar/SideNavBarItemGroup.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component, PropTypes } from 'react';
+import { forEach, find } from 'lodash';
+import SideNavBarItem from './SideNavBarItem';
+import style from './style.scss';
+
+class SideNavBarItemGroup extends Component {
+
+  constructor() {
+    super();
+    this.handleGroupClick = this.handleGroupClick.bind(this);
+    this.handleItemClick = this.handleItemClick.bind(this);
+  }
+
+  handleGroupClick() {
+    if (this.props.items && this.props.items[0]) {
+      this.props.onClick({ id: this.props.items[0].id });
+    } else {
+      this.props.onClick({ id: this.props.id });
+    }
+  }
+
+  handleItemClick(itemId) {
+    this.props.onClick({ id: itemId });
+  }
+
+  renderIconAndLabel() {
+    const { icon, iconSize, iconStyle, label, items } = this.props;
+    if (!find(items, (item) => !item.hidden)) {
+      return null;
+    }
+
+    const renderIcon = icon && iconSize && (
```

Comment:
minor: not to sound pedantic, but renderIcon sounds like a method name and kinda throws you off when reading code; can we do smth like `iconToRender`?